javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,161 @@
+/*
+***************************************************************************************************************************************
+                                                         Voice assistant     
+***************************************************************************************************************************************
+*/
+let userName = "";
+let todolist = [];
+
+function getReply(command) {
+  let tempArray = command.split(" ");
+
+  //get name and reply with the user's name
```

Comment:
Super nice that you added the comments. Though these are only small tasks and you won't need them here, but a good practice I would say 👍